src/utils/pricing_utils.ts

```diff
@@ -97,9 +100,23 @@ export class PlanLimitsHandler {
 			if (this.repoId) return await this.run();
 			// This is "Connect Repo"
 			const isNewPrivateRepo = errorCode === ErrorCodes.PRIVATE_REPO_COUNT_LIMIT_REACHED;
-			showFreeTierLimitReached(this.repoPath, isNewPrivateRepo);
+			const canAvailTrial = await getCanAwailTrial(this.accessToken);
+			showFreeTierLimitReached(this.repoPath, isNewPrivateRepo, canAvailTrial);
```

Comment:
Can we move 
```const canAvailTrial = await getCanAwailTrial(this.accessToken);```

inside the `showFreeTierLimitReached`

That way API call will also be tested when we test `showFreeTierLimitReached`

Reply:
Did I get that right?

Reply:
That's correct